components/common/FloatingRibbon.tsx

```diff
@@ -1,16 +1,18 @@
 import { Button as BaseButton, Center } from '@chakra-ui/react'
+import { RESPONSIVE_PADDING } from './ContentBox'
 
 export default function FloatingRibbon({ children }) {
   return (
     <Center
-      pos='sticky'
+      pos="sticky"
       bottom={0}
       left={0}
       right={0}
-      p={4}
-      borderTopRadius='8px'
-      boxShadow='0px -4px 4px rgba(0, 0, 0, 0.1)'
-      bg='white'
+      py={4}
+      px={RESPONSIVE_PADDING}
```

Comment:
Lots of reformatting by prettier in this one. Main change is here: `px={RESPONSIVE_PADDING}`